models/NeatlineExhibit.php

```diff
@@ -82,8 +106,15 @@ public function getNumberOfRecords()
     public function getRecordUrl($action = 'show')
     {
         $urlHelper = new Omeka_View_Helper_Url;
-        $params = array('action' => $action, 'id' => $this->id);
-        return $urlHelper->url($params, 'neatlineActionId');
+        $checkUrl = $urlHelper->url();
+        if (strpos($checkUrl, '/admin/') !== false) {
```

Comment:
Thanks @jamiefolsom — that's definitely a non-ideal fallback there, and I'll give it another look. My impression on first pass was that the proper way to determine whether the call was coming from inside an admin controller/view would be to check the `view` property of the $urlHelper object, but it seems to be left as null in this case. Hopefully some further digging into other plugin repos will turn up a better workaround.